mindmeld/app.py

```diff
@@ -171,6 +172,20 @@ def _decorator(func):
 
         return _decorator
 
+    def auto_fill(self, **kwargs):
+        """Creates a flow to fill missing entities"""
+
+        def _decorator(func):
+            name = kwargs.pop("name", func.__name__)
+            entity_form = kwargs.pop("entity_form", None)
```

Comment:
You can move this argument to be explicitly required in the decorator signature.  This will enforce the check at parse time rather than runtime as well.

`def autofill(self, max_retries=1, *, entity_form=None, **kwargs):
`

Disclaimer: I didn't syntax check this ^^, but something along these lines should do it.